### PR TITLE
Meisi branch

### PR DESCRIPTION
**Pull request by huangmeisi**
this is a dummy change.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,7 @@
 start:
 	docker-compose up
 
+# change1
```
Comment (qiqiustb) [inline Makefile:4]: this is not right.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 Slack Challenge
 ===============
+Change 2
```
Comment (qiqiustb) [inline README.md:3]: this is also not right.

Comment (huangmeisi) [reply README.md:3]: ok then.

File: README.md [hunk 3]
```diff
@@ -11,6 +11,7 @@ It's currently hosted on an AWS EC2 box: http://ec2-54-70-70-30.us-west-2.comput
 2. visit http://yourhostname
 
 #### How does this app work ?
+change 3
```
Comment (qiqiustb) [inline README.md:14]: blabla.

**Review by qiqiustb — CHANGES_REQUESTED** [review]
I request for a change.

File: README.md (hunk 3 above)
Comment (huangmeisi) [reply README.md:14]: haode

**Review by qiqiustb — APPROVED** [review]
LGTM

**Pull request merged**